### ISSUE TITLE
Split the package into small packages

### ISSUE THREAD

**Issue by jmg-duarte** [sbdchd/celery-types#74]
👋 Hello, I am interested in the `kombu` stubs and nothing else. 

From what I saw, there is no way to simply use the `kombu` typings, I need to download everything.

What are your thoughts on publishing each package individually and make the current repository an aggregation of the individual parts?

**Comment by sbdchd** [comment]
Works for me!

**Comment by 9en9i** [comment]
Yes, in fact, dividing the package into several parts will require additional support resources. Could you close this? @sbdchd
